src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
It's good that you've used a specific font for your website, but remember to use fallback fonts. Fallback fonts are used in case the main font doesn't work. For example, you could modify this line to be `font-family: Roboto, Arial, sans-serif, serif;` Here, `serif` is the fallback font.